.rubocop.yml

```diff
@@ -1,14 +1,15 @@
 require: 'rubocop-rspec'
 
 AllCops:
-  TargetRubyVersion: 2.3
+  NewCops: enable
+  TargetRubyVersion: 2.6.9
```

Comment:
```suggestion
  TargetRubyVersion: 2.6
```

I don't think the patch version will do anything?

Reply:
🤷 it didn't complain 😅